pkg/controller/seed-controller-manager/encryption-at-rest-controller/controller.go

```diff
@@ -177,9 +179,62 @@ func (r *Reconciler) Reconcile(ctx context.Context, request reconcile.Request) (
 	return *result, err
 }
 
+func getSecretKeyValue(client ctrlruntimeclient.Client, ref *corev1.SecretKeySelector, namespace string) ([]byte, error) {
+	secret := corev1.Secret{}
+	if err := client.Get(context.Background(), ctrlruntimeclient.ObjectKey{
```

Comment:
This should take a `ctx context.Context` argument and pass that to the client calls.